ros/config/basic_navigation_params.yaml

```diff
@@ -1,7 +1,8 @@
 global_frame: 'map'
+robot_frame: 'load/base_link'
 use_global_planner: True
 num_of_retries: 5
-allow_backward_motion: True
+allow_backward_motion: False
```

Comment:
Having this set to `False` means that the robot will never move back, right? What we need is to be able to switch the parameters on the fly, just like for the maneuver and napoleon navigation.

Reply:
It is a ros parameter. I can make it such that the robot reads this ros parameter every time it receives a new goal (instead of just reading once in `__init__`). This way you can set the ros param and send the goal. Will that be alright?